accountsplus/validators.py

```diff
@@ -14,3 +14,15 @@ def validate(self, password, user=None):
     def get_help_text(self):
          return _('Password should contain uppercase, lowercase, numeric values and at least '
                   'one of the following $@#!%*?&')
+
+
+class CustomPasswordValidator(object):
+
+    def validate(self, password, user=None):
+        regex = '(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[A-Za-z\d]'
```

Comment:
@nasief will this regex allow special characters? have you tested it locally?

Reply:
@waseem-omar Yes I did. The validator here only makes restrictions related to lowercase, uppercase and digits other characters are allowed.